tests/hardhat/util/Proposals.ts

```diff
@@ -0,0 +1,16 @@
+export enum ProposalType  {
+	NORMAL,
+	FASTTRACK,
+	CRITICAL
+}
+
+export enum ProposalState  {
+	Pending,
+	Active,
+	Canceled,
+	Defeated,
+	Succeeded,
+	Queued,
+	Expired,
+	Executed
+}
```

Comment:
We should use a linting rule that ensures a new line at the end of every file.

Reply:
Yeah we need to setup a global lint for the projects